src/aiidalab_qe/app/wizard_app.py

```diff
@@ -131,12 +128,17 @@ def _on_step_change(self, change):
 
     def _on_structure_confirmation_change(self, _):
         self._update_configuration_step()
+        if not self.process:
+            self._wizard_app_widget.selected_index = 1
```

Comment:
Instead of hard-coding the index, better to increase the index by 1. Same for others.
```suggestion
            self._wizard_app_widget.selected_index = self._wizard_app_widget.selected_index + 1
```